### PR TITLE
Missing listen feature added 

### LINKED ISSUE #569: The page loses functionality while fetching data
The information displayed regarding cities under ```Know more about``` in the application is fetched form wikimedia that uses the text to speech feature for the city names pronounciation. But in the application the feature is missing and only ```listen``` word is displayed without the sound feature.  

### Steps to reproduce
Choose cities like Bangalore, Chennai, Kolkata, Jaipur, Hyderabad, Ahmedabad and when we click to ```know more about``` these cities we see  *(listen)*  word displayed without the pronunciation functionality available. 

### Expected behaviour
It should have the pronunciation available when the user click on ```listen```  

### Actual behaviour
Only ```listen``` word is displayed without any pronunciation .

### Would you like to work on the issue?
Yes

### PR DESCRIPTION
**Pull request by ShivangiSingh17**
## Description
The audio clip was not playing while the data was fetched from the wiki pages. Now the audio files are added that will play the audio file when the user opts for the pronunciation of the word by clicking on ```listen```

Fixes #569 

## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [X] `./gradlew assembleDebug assembleRelease`
- [X] `./gradlew checkstyle`

# Checklist:
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings

### CODE REVIEW

File: .idea/Travel-Mate.iml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Swati4star) [inline .idea/Travel-Mate.iml]: remove .idea folder from commit

File: .travis.yml [hunk 2]
```diff
@@ -13,6 +14,8 @@ android:
     - extra-google-m2repository
   licenses:
     - '.+'
+before_install:
+  - chmod +x gradlew
```
Comment (Swati4star) [inline .travis.yml]: are these changes required? If so, why?

File: Android/app/src/main/java/utils/listenaudio.java [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+public class ColorsActivity extends AppCompatActivity {
+
+    /** Handles playback of all the sound files */
+    private MediaPlayer mMediaPlayer;
+
+    /** Handles audio focus when playing a sound file */
+    private AudioManager mAudioManager;
+
+    /**
+     * This listener gets triggered when the {@link MediaPlayer} has completed
```
Comment (Swati4star) [inline Android/app/src/main/java/utils/listenaudio.java]: add javadoc style comments

**Comment by ShivangiSingh17** [comment]
> Do we have to add audio files for each state in this case? Is there no other way?
> 
> Also, trvais checks are failing

I tried with removing the whole functionality but that did not work so I approached in this way.
Do you have any suggestion to approach, then please do guide me.

File: .travis.yml (hunk 2 above)
Comment (ShivangiSingh17) [reply .travis.yml]: The Travis was failing so I referred to a post and made changes to solve but still the problem was nort solved.

File: Android/app/src/main/java/utils/listenaudio.java (hunk 3 above)
Comment (ShivangiSingh17) [reply Android/app/src/main/java/utils/listenaudio.java]: ok I will do that
